ci/build-check.sh

```diff
@@ -6,7 +6,9 @@ set -xeuo pipefail
 dn=$(dirname $0)
 . ${dn}/libbuild.sh
 ${dn}/build.sh
-make check
+# NB: override parallel builds because our RPM building
+# doesn't # support that right now
+make check -j1
```

Comment:
I'd say `/usr/bin/make check` rather; not sure offhand about the semantics of multiple `-j` args.